udata/harvest/actions.py

```diff
@@ -162,3 +164,43 @@ def unschedule(ident):
     source.periodic_task.delete()
     signals.harvest_source_unscheduled.send(source)
     return source
+
+
+def attach(domain, filename):
+    '''Attach existing dataset to their harvest remote id before harvesting.
+
+    The expected csv file format is the following:
+
+    - a column with header "local" and the local IDs or slugs
+    - a column with header "remote" and the remote IDs
+
+    The delimiter should be ";". columns order
+    and extras columns does not matter
+    '''
+    count = 0
+    with open(filename) as csvfile:
```

Comment:
It's only IDs, no unicode but your right, maybe I should use unicodecsv here too.